test/inference/Inference.jl

```diff
@@ -131,4 +131,12 @@
         @test mean(x[:s][1] for x in chains) ≈ 3 atol=0.1
         @test mean(x[:m][1] for x in chains) ≈ 0 atol=0.1
     end
+
+    @testset "chain ordering" begin
+        chain_sorted = sample(gdemo_default, Prior(), 1, sort_chain=true)
+        @test names(MCMCChains.get_sections(chain_sorted, :parameters)) == [:m, :s]
+
+        chain_unsorted = sample(gdemo_default, Prior(), 1, sort_chain=false)
+        @test names(MCMCChains.get_sections(chain_unsorted, :parameters)) == [:m, :s]
```

Comment:
Not completely surprising, this test fails. I guess it should be
```suggestion
        @test names(MCMCChains.get_sections(chain_unsorted, :parameters)) == [:s, :m]
```
?

Reply:
BTW it might be good to check the output without keyword argument as well, to check that the default value of `sort_chains` is set as expected.

Reply:
One additional thought: maybe `sort_parameters` would be more explicit since the chain is a 3D array and hence in principle could be sorted with respect to different axes?

Reply:
Ehh yes:)